release-tools/build.make

```diff
@@ -101,9 +106,42 @@ push-%: container-%
 		fi; \
 	done
 
+RELEASE_ALIAS_TAG=$(PULL_BASE_REF)
+
+push-multiarch-%:
```

Comment:
This whole target needs documentation. A short explanation of how it works and what the expected environment is would be useful.

Reply:
Added short explanation about this target